javers-spring-boot-starter-mongo/src/main/java/org/javers/spring/boot/mongo/JaversMongoAutoConfiguration.java

```diff
@@ -62,13 +62,15 @@ public Javers javers() {
     }
 
     @Bean(name = "authorProvider")
+    @ConditionalOnMissingBean
```

Comment:
Sytuacja wygląda tak, że mamy dwa konkurencyjne beany aplikowane na podstawie obecności lub nieobecności klasy na classpathie. Jeżeli ustawiona jest tylko adnotoacja @ConditionalOnBean na jednym z tych beanów to faktycznie całość może to działać niedeterministycznie. Jak rozumiem trzeba dodać na drugim z nich @ConditionalOnMissingBean żeby warunki się wykluczały i ten commit Kuby właśnie to robi (tytuł commita to: "deterministic AuthorProvider configuration")